study-builder/studies/mpc/study-activities.conf

```diff
@@ -0,0 +1,63 @@
+{
+  "activityStatusIcons": [
+    {
+      "filepath": "icons/created.svg",
+      "statusType": "CREATED"
+    },
+    {
+      "filepath": "icons/in_progress.svg",
+      "statusType": "IN_PROGRESS"
+    },
+    {
+      "filepath": "icons/complete.svg",
+      "statusType": "COMPLETE"
+    }
+  ],
+  "activityTimestamp": "2015-07-01T00:00:00Z",
```

Comment:
I understand that we have not put in the Consent V1 yet. However, this will not do what you want. Note that adding this `activityTimestamp` means all the activities that will be created in a study-builder run will have this starting timestamp. This means the Consent V2 activity we have here will be in year 2015 when what we want is year 2019.

I think it will be less confusing if we start off with Consent V1 in the `conf` file and insert the V2 after that.